src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace kOS.Suffixed
 {
-    public class OrbitInfo : Structure, IDumperWithSharedObjects
+    public class OrbitInfo : SerializableStructure, IHasSharedObjects
```

Comment:
The reason I noticed it in this PR is that this PR is the first time the code explicitly claimed this was derived from SerializableStructure, which was a class that didn't exist before this PR.  But conceptually, this is not really a serializable class, because of this.